traffic_ops/app/bin/checks/DnssecRefresh/config/config.go

```diff
@@ -0,0 +1,153 @@
+	logLocationInfoPtr := getopt.StringLong("log-location-info", 'i', "stderr", "Where to log info. May be a file path, stdout, stderr, or null, default stderr")
+	logLocationWarnPtr := getopt.StringLong("log-location-warning", 'w', "stderr", "Where to log warnings. May be a file path, stdout, stderr, or null, default stderr")
+	toInsecurePtr := getopt.BoolLong("traffic-ops-insecure", 'I', "[true | false] ignore certificate errors from Traffic Ops")
+	toUserPtr := getopt.StringLong("traffic-ops-user", 'u', "", "Traffic Ops username. Required.")
+	toPassPtr := getopt.StringLong("traffic-ops-passowrd", 'p', "", "Traffic Ops Password. Required")
+	toUrlPtr := getopt.StringLong("traffic-ops-url", 'U', "", "Traffic ops base URL. Required.")
+	helpPtr := getopt.BoolLong("help", 'h', "Print usage information and exit")
+	getopt.ParseV2()
+
+	logLocationDebug := *logLocationDebugPtr
+	logLocationError := *logLocationErrorPtr
+	logLocationInfo := *logLocationInfoPtr
+	logLocationWarn := *logLocationWarnPtr
+	toInsecure := *toInsecurePtr
+	toURL := *toUrlPtr
+	toUser := *toUserPtr
+	toPass := *toPassPtr
+	transport := &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: toInsecure}}
+	help := *helpPtr
+
+	if help {
+		Usage()
+		return Cfg{}, nil
+	}
+
+	missingArgStr := "Missing required argument"
+	usageStr := "\nBasic usage: ToDnssecRefresh --traffic-ops-url=myurl --traffic-ops-user=myuser --traffic-ops-password=mypass"
+	if strings.TrimSpace(toURL) == "" {
+		return Cfg{}, errors.New(missingArgStr + " --traffic-ops-url" + usageStr)
+	}
+	if strings.TrimSpace(toUser) == "" {
+		return Cfg{}, errors.New(missingArgStr + " --traffic-ops-user" + usageStr)
+	}
+	if strings.TrimSpace(toPass) == "" {
+		return Cfg{}, errors.New(missingArgStr + " --traffic-ops-password" + usageStr)
+	}
+
+	cfg := Cfg{
+		LogLocationDebug: logLocationDebug,
+		LogLocationErr:   logLocationError,
+		LogLocationInfo:  logLocationInfo,
+		LogLocationWarn:  logLocationWarn,
+		TOInsecure:       toInsecure,
+		Transport:        transport,
+		TOUrl:            toURL,
+		TOUser:           toUser,
+		TOPass:           toPass,
+	}
+
+	if err = log.InitCfg(cfg); err != nil {
+		return Cfg{}, errors.New("Initializing loggers: " + err.Error() + "\n")
+	}
+
+	return cfg, nil
+}
+
+func PrintConfig(cfg Cfg) {
+	log.Debugf("TOUrl: %s\n", cfg.TOUrl)
+	log.Debugf("TOUser: %s\n", cfg.TOUser)
+	log.Debugf("TOPass: Pass len: %d\n", len(cfg.TOPass))
+	log.Debugf("TOInsecure: %t\n", cfg.TOInsecure)
+	log.Debugf("LogLocationDebug: %s\n", cfg.LogLocationDebug)
+	log.Debugf("LogLocationErr: %s\n", cfg.LogLocationErr)
+	log.Debugf("LogLocationInfo: %s\n", cfg.LogLocationInfo)
+	log.Debugf("LogLocationWarn: %s\n", cfg.LogLocationWarn)
+}
+
+func Usage() {
+	fmt.Println("\t  --log-location-debug=[value] | -d [value], Where to log debugs. May be a file path, stdout, stderr, or null, default stderr")
```

Comment:
Nitpick: this might be easier to read as a raw string literal:
```go
usageStr := `	--log-location-debug=[value] | -d [value], Where to log debugs. May be a file path, stdout, stderr, or null, default stderr
	--log-location-error=[value] | -e [value], Where to log errors. May be a file path, stdout, stderr, or null, default stderr
	--log-location-info=[value] | -i [value], Where to log info. May be a file path, stdout, stderr, or null, default stderr
	--log-location-warning=[value] | -w [value], Where to log warnings. May be a file path, stdout, stderr, or null, default stderr
	--traffic-ops-url=[url] | -u [url], Traffic Ops URL. Must be the full URL, including the scheme. Required."
	--traffic-ops-user=[username] | -U [username], Traffic Ops username. Required."
	--traffic-ops-password=[password] | -P [password], Traffic Ops password. Required.
	--help | -h, Print usage information and exit`
	fmt.Println(usageStr)
```

Reply:
Changed to raw string literal